src/Mappers/Loyalty/Receptions/DigitalRewardReceptionMapper.php

```diff
@@ -23,8 +23,16 @@ public function map(stdClass $data): DigitalRewardReception
 
         $contact = $contactMapper->map($data->contact);
         $shop = $shopMapper->map($data->shop);
-        $digitalReward = $digitalRewardMapper->map($data->digital_reward);
-        $digitalRewardCode = $digitalRewardCodeMapper->map($data->digital_reward_code);
+
+        $digitalReward = null;
```

Comment:
It's imo nicer to remove line 27 & 32 (`$digitalReward = null` and `$digitalRewardCode = null`), and to then do the following:

```php
        return new DigitalRewardReception(
            // Other properties...
            $digitalReward ?? null,
            $digitalRewardCode ?? null
);
```